XRTK-Core/Packages/com.xrtk.core/Editor/Profiles/MixedRealityToolkitRootProfileInspector.cs

```diff
@@ -21,9 +21,6 @@ namespace XRTK.Editor.Profiles
     [CustomEditor(typeof(MixedRealityToolkitRootProfile))]
     public class MixedRealityToolkitRootProfileInspector : MixedRealityServiceProfileInspector
     {
-        // Additional registered components profile
-        private SerializedProperty registeredServiceProvidersProfile;
```

Comment:
I would like to put this back in please. That way there's a UI to register services that aren't systems

Reply:
Now this I don't get.

We remove Extension Services, so there is a single unified collection of Services/Systems through the inspector.

But now you want to retain the separate profile we were removing so "other" services can be configured.

Surely we should have a better plan where ALL services are in a single place and not hidden behind other config

Reply:
@SimonDarksideJ I think I understand what Stephen is going for. Say you remove that profile, then in order to to register any custom services, I'll have to first create a "dummy" system and register it in the root profile. Only then I will be able to add/register services to that system via the inspector.

So basically this `RegisteredServiceProvidersProfile` is that "dummy" system. It allows me to register services without having to worry abotu setting up a system first. Does that make sense to you?

Reply:
Remember system = only one active implementation/instance. Service = multiple instances allowed.

Reply:
I think you did a great job removing the extension services defintions. As we don't need them. You united all services to be the same, which is a win. But I do also believe we can keep this profile just so people have a way to register services without having to worry about setting up a system first.

Reply:
Which is fair enough, but as you can have a system that can have MANY data providers, isn't that a better approach than multiple services?  Using more of a "Pool" approach.

Reply:
Remember this diagram?
![0_MTUg71o438oMaOBv](https://user-images.githubusercontent.com/13334553/128195182-80946670-5698-4051-b2d0-7656ce7371d3.png)